cache/README.md

```diff
@@ -33,7 +33,7 @@ Let's add this to a new file in `client-mu-plugins` called `beta.php` where we'l
 <?php
 
 // Load the VIP Vary_Cache class
-require_once( WPMU_PLUGIN_DIR . '/cache/class-vary-cache.php' );
+require_once( WPVIP_MU_PLUGIN_DIR . '/cache/class-vary-cache.php' );
 
```

Comment:
Please note that this is going to have to be something we account for in the migration. Any customer code currently doing things like this will break.

Reply:
Good point.